jcommune-model/src/test/java/org/jtalks/jcommune/model/dao/hibernate/GroupHibernateDaoTest.java

```diff
@@ -101,6 +104,12 @@ public void testGetAll() {
         saveAndEvict(group1);
 
         List<Group> actual = groupDao.getAll();
+        Collections.sort(actual, new Comparator<Group>() {
```

Comment:
To keep tests declarative, could you please move this into a separate method like `sortById(List<Group>)`?